packages/wonder-blocks-core/src/index.js

```diff
@@ -12,6 +12,7 @@ export {
     useUniqueIdWithMock,
     useUniqueIdWithoutMock,
 } from "./hooks/use-unique-id.js";
+export {useForceUpdate} from "./hooks/use-force-update.js";
```

Comment:
There should probably be a storybook documentation page for this hook since it's part of the public API.

Reply:
I have a separate task for the docs work.

Reply:
See FEI-4191